### DISCUSSION TITLE
second EM3 inverter

### DISCUSSION THREAD

**Discussion by Daniel-Peeeh** [fabian-lauer/dbus-shelly-3em-smartmeter#18] [Q&A]
Hi, great project and many thanks! First setup with EM3 Smart meter up and running. 
Now there is one question: Is it possible to connect another EM3 to Venus OS giving the measurement of an inverter which is not able to communicate directly with the system?

thanks in advance

**Comment by fabian-lauer** [comment]
Hi @Daniel-Peeeh , you may checkout this pull-request: https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/pull/14
I need a bit more time to check it, but this looks a bit like what you are looking for.

**Comment by kptkip** [comment]
Would be a great idea! :.-)

**Comment by gartenjoe** [comment]
Hello Daniel, I need it for exactly the same purpose. 2 Steca hybrid inverters that cannot be connected. That's why I wanted to measure them single-phase with Shellys and have them displayed in Venus. Greetings Detlev

**Comment by Daniel-Peeeh** [comment]
Detlev,
after more research I also found this. for your single phase measurement you are using the 1pm i guess.
https://github.com/vikt0rm/dbus-shelly-1pm-pvinverter

I think it`s exactly what you are searching for...

**Comment by mo0815** [comment]
ola yess change the name of the one allready running by changing the Name
also the service in rc has to be changed 
aswell the service/run file name
